templates/vue/copy-dev-account.js

```diff
@@ -0,0 +1,7 @@
+let fs=require('fs')
```

Comment:
You can make this file executable (`chmod +x copy-dev-account.js`) if you add one line to the top:

    #!/usr/bin/env node

Then you could drop the `.js` from the filename and invoke it with `./copy-dev-account` instead of `node copy-dev-account`

Up to you. I like this pattern 🙂

Reply:
Not insisting, I like your reasoning 😄